src/table/body-cell/styles.scss

```diff
@@ -96,3 +105,19 @@ $border-placeholder: awsui.$border-item-width solid transparent;
     padding-top: $cell-vertical-padding;
   }
 }
+
+/*
+In Visual Refresh the first cell of each row should align with the 
+left edge of the table.
+*/
+.body-cell:not(.is-visual-refresh) {
+  &:first-child {
+    padding-left: $cell-edge-horizontal-padding;
+  }
+}
+
+.body-cell.is-visual-refresh {
+  &:first-child {
+    padding-left: 0;
```

Comment:
This seems to affect also the full page variant. Should this one keep a left padding? See `/#/light/table/full-page-variant/`

![Screenshot 2022-12-08 at 11 07 01](https://user-images.githubusercontent.com/1257272/206419173-7da6e338-edcb-410d-8f75-0ac3019ced5c.png)

Reply:
Good find! This is intentional as the full page variant has no internal padding and is assumed to be used exclusively within the `AppLayout` component (with `contentType='table'`). The same behavior can be seen with the `Wizard` component. We are having some internal discussions on how to better handle our full page patterns but for the purposes of this PR this is correct.